keystrokes/src/components/Auth/auth.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { auth } from '../../config/constants';
+import Home from './home';
+import Login from './login';
+
+class Auth extends React.Component {
+	constructor(props) {
+		super(props);
+		this.authListener = this.authListener.bind(this);
```

Comment:
Because this.authListener is called in `this` Component you don't need to bind it. 
if you passed it to a child component or had it as an event listener and needed access to `this` then you'd need to bind it.